safety_limiter/test/src/test_safety_limiter.cpp

```diff
@@ -559,6 +559,59 @@ TEST_F(SafetyLimiterTest, SafetyLimitLinearSimpleSimulation)
   }
 }
 
+TEST_F(SafetyLimiterTest, SafetyLimitMaxVelocitiesValues)
+{
+  ros::Rate wait(20);
+
+  const float linear_velocities[] =
+  {-1.7, -1.5, 0.0, 1.5, 1.7 };
+  const float angular_velocities[] =
+  {-2.7, -2.5, 0.0, 2.5, 2.7 };
+  const float expected_linear_velocities[] =
+  {-1.5, -1.5, 0.0, 1.5, 1.5 };
+  const float expected_angular_velocities[] =
+  {-2.5, -2.5, 0.0, 2.5, 2.5 };
+
+  for (int linear_index = 0; linear_index < 5; linear_index++)
+  {
+    for  (int angular_index = 0; angular_index < 5; angular_index++)
+    {
+      bool received = false;
+      bool en = false;
+
+      for (int i = 0; i < 10 && ros::ok(); ++i)
+      {
+        if (i > 5) en = true;
+        publishSinglePointPointcloud2(1000, 1000, 0, "base_link", ros::Time::now());
+        publishWatchdogReset();
+        publishTwist(linear_velocities[linear_index], angular_velocities[angular_index]);
+        broadcastTF("odom", "base_link", 0.0, 0.0);
+
+        wait.sleep();
+        ros::spinOnce();
+        if (en && cmd_vel_)
+        {
+          received = true;
+          ASSERT_NEAR(cmd_vel_->linear.x, expected_linear_velocities[linear_index], 1e-3);
```

Comment:
gtest's argument order is `(expected, actual)`

Reply:
My bad :|
Fixed